### PR TITLE
gh-2142 Removed 3 unused imports

### PR DESCRIPTION
**Pull request by w122stern**
#2142

### CODE REVIEW

File: core/operation/src/test/java/uk/gov/gchq/gaffer/operation/impl/JoinTest.java [hunk 1]
```diff
@@ -17,11 +17,8 @@
 package uk.gov.gchq.gaffer.operation.impl;
 
 import com.google.common.collect.Lists;
-import com.sun.javafx.binding.StringFormatter;
 import org.apache.commons.lang.StringUtils;
```
Comment (m55624) [inline core/operation/src/test/java/uk/gov/gchq/gaffer/operation/impl/JoinTest.java]: Can you add a new line after these.

File: core/operation/src/test/java/uk/gov/gchq/gaffer/operation/impl/JoinTest.java [hunk 2]
```diff
@@ -17,11 +17,8 @@
...
-import uk.gov.gchq.gaffer.commonutil.StringUtil;
 import uk.gov.gchq.gaffer.data.element.Entity;
-import uk.gov.gchq.gaffer.jsonserialisation.JSONSerialiser;
 import uk.gov.gchq.gaffer.operation.OperationTest;
 import uk.gov.gchq.gaffer.operation.impl.get.GetAllElements;
 import uk.gov.gchq.gaffer.operation.impl.join.Join;
```
Comment (m55624) [inline core/operation/src/test/java/uk/gov/gchq/gaffer/operation/impl/JoinTest.java]: Also whilst we are on it, could you update the starred import below to this:
import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertNotEquals;
import static org.junit.Assert.assertSame;
import static org.junit.Assert.assertTrue;

**Comment by w122stern** [comment]
Thanks for the clarification, I have made the changes.

File: core/operation/src/test/java/uk/gov/gchq/gaffer/operation/impl/JoinTest.java [hunk 3]
```diff
@@ -46,7 +47,8 @@ public void builderShouldCreatePopulatedOperation() {
                 .joinType(JoinType.INNER)
                 .flatten(false)
                 .collectionLimit(10)
-                .build();;
+                .build();
+        ;
```
Comment (d47853) [inline core/operation/src/test/java/uk/gov/gchq/gaffer/operation/impl/JoinTest.java]:
```suggestion
```

**Pull request merged**